### DISCUSSION TITLE
Is there an Easier way to achieve Desktop First responsive?

### DISCUSSION THREAD

**Discussion by someryang** [chakra-ui/chakra-ui#7061] [Q&A]
I want to change the default breakpoints config in chakra to Desktop first.
in Tailwind I use screens :
screens: {
			laptop: { max: "1023px" },
			// => @media (max-width: 1023px) { ... }
			tablet: { max: "767px" },
			// => @media (max-width: 767px) { ... }
			mobile: { max: "639px" },
			// => @media (max-width: 639px) { ... }
		},
but I dont know how to achive this,
I check the method in https://ethereum.org/en/ this webSite,
But I think this is too complex.
Is there an Easier way to achieve Desktop First responsive?

**Comment by wrux** [comment]
It doesn't look like it. 

ChakraUI internally uses a function called [toMediaQueryString](https://github.com/chakra-ui/chakra-ui/blob/main/packages/utilities/breakpoint-utils/src/breakpoint.ts#L48) which hardcodes `min-width:` into the media query
